dallinger/command_line/docker_ssh.py

```diff
@@ -781,7 +788,7 @@ def destroy(server, app):
 class Executor:
     """Execute remote commands using paramiko"""
 
-    def __init__(self, host, user=None, app=None):
+    def __init__(self, host, user=None, app=None, server_pem=None):
```

Comment:
Likewise I think it would probably make sense to remove the server_pem argument and just query config inside `Executor.__init__`? That way you don't need to worry about writing extra code every time `Executor` is instantiated.

Reply:
See comment above regarding refactoring.